dir.c

```diff
@@ -3345,7 +3345,7 @@ rb_file_directory_p(void)
 static VALUE
 rb_dir_exists_p(VALUE obj, VALUE fname)
 {
-    rb_warn_deprecated("Dir.exists?", "Dir.exist?");
+    rb_warn_deprecated_to_remove_use_this_one_instead("Dir.exists?", "3.2", "Dir.exist?");
```

Comment:
Why is only this 3.2?

Reply:
@nobu Read #3412 discussion.

Reply:
I should have asked about `FileTest.exists?` in the table at this page.
As that method is to be removed at 3.2 in the code, just the table is outdated a little?

Reply:
I missed that the table updated in [rev. 2](https://github.com/ruby/ruby/pull/3412)...